packages/rancher-istio/charts/README.md

```diff
@@ -57,13 +69,58 @@ http://tracing.{{ .Values.namespaceOverride }}.svc:{{ .Values.service.externalPo
 ```
 The url depends on the default values for `namespaceOverride`, and `.Values.service.externalPort` being set in your rancher-tracing or other tracing instance.
 
-## Jaeger
+## Jaeger Addon
 
 Jaeger allows you to trace and monitor distributed microservices.
 
 > **Note:** This addon is using the all-in-one Jaeger installation which is not qualified for production. Use the [Jaeger Tracing](https://www.jaegertracing.io/docs/1.21/getting-started/) documentation to determine which installation you will need for your production needs.
 
-# Installation
+## Known Issues
+
+#### Airgapped Environments
+If you are using this chart in an airgapped environment, you will not be able to upgrade. This is because the `istioctl` upgrade command reaches out to an external repo and it is not configurable. We are tracking the fix for this issue [here](https://github.com/rancher/rancher/issues/33402)
+
+## Chart Deprecations
+
+#### v1alpha1 security policies
```

Comment:
This is a ton of info, let's either post this in the rancher docs or the issue, and just link out to them. We want to keep the app-readme clean and small. If istio has these docs available we could also use those.

Reply:
this has been cleaned up in the most recent commits.